besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2628,4 +2630,23 @@ public int getDatabaseVersion() {
           .getDatabaseVersion();
     }
   }
+
+  private void instantiateSignatureAlgorithmFactory() {
+    if (SignatureAlgorithmFactory.isInstanceSet()) {
+      return;
+    }
+
+    if (genesisFile == null) {
+      SignatureAlgorithmFactory.setInstance(SignatureAlgorithmType.create(Optional.empty()));
```

Comment:
Will that use the default key algorith? We could simplify this code a bit adding a method to the SignatureAlgorithmFactory like:
```
SignatureAlgorithmFactory.setDefault();
```
Makes it easier to read.

Reply:
Actually, do we even need to set anything? SignatureAlgorithFactory.getInstance() has a default value, hasn't it?

Reply:
You are right, we don't need to set anything here. I have replaced  as well the type of `Optional<String>` simply by `String`. That makes the code a bit easier to read.